mailpoet/tests/acceptance/Settings/WooCommerceSettingsTabCest.php

```diff
@@ -15,11 +18,58 @@ class WooCommerceSettingsTabCest {
   /** @var Settings */
   private $settingsFactory;
 
+  /** @var array WooCommerce Product data*/
+  private $product;
+  private $segmentName;
```

Comment:
It is a tiny detail, but it would be nice to hint to the editor that the segment name is a string.

```suggestion

  /** @var string */
  private $segmentName;
```

Reply:
Thanks for the suggestion, done.